riscv-syscalls/src/lib.rs

```diff
@@ -54,4 +54,6 @@ syscalls!(
     (5, EcAdd, "ec_add"),
     (6, EcDouble, "ec_double"),
     (7, KeccakF, "keccakf"),
+    (8, Mod256, "mod_256"),
+    (9, Halt, "halt"),
```

Comment:
These 2 seem unrelated to this PR?

Reply:
Yeah. These are just changes from merge main and I think it appears as additions because you set review display to the latest 6 commits. They won't appear as changes if you display all commits.